packages/samples/headless-react/src/pages/AtomicReactPage.tsx

```diff
@@ -4,41 +4,207 @@ import {
+      engine={engine}
+      pipeline="Search"
+      searchHub="MainSearch"
+    >
+      <div className="search">
+        <AtomicSearchBox />
+      </div>
+      <AtomicFacetManager>
+        <AtomicCategoryFacet field="ec_category" label="Category" withSearch />
+        <AtomicFacet field="ec_brand" label="Brand" />
+        <AtomicNumericFacet
+          field="cat_review_count"
+          label="Amount of reviews"
+          displayValuesAs="link"
         >
-          <AtomicResultSectionBadges>
-            <AtomicResultBadge field="source"></AtomicResultBadge>
-          </AtomicResultSectionBadges>
-          <AtomicResultSectionTitle>
-            <AtomicResultLink></AtomicResultLink>
-          </AtomicResultSectionTitle>
-        </AtomicResultTemplate>
-      </AtomicResultList>
+          <AtomicNumericRange start={0} end={150} label="Few" />
+          <AtomicNumericRange start={150} end={650} label="A moderate amount" />
+          <AtomicNumericRange start={650} end={9999999999} label="A lot" />
+        </AtomicNumericFacet>
+        <AtomicColorFacet
+          field="cat_color"
+          label="Color"
+          numberOfValues={6}
+          sortCriteria="occurrences"
+        />
+        <AtomicNumericFacet field="ec_price" label="Cost" withInput="integer">
+          <AtomicFormatCurrency currency="USD" />
+        </AtomicNumericFacet>
+        <AtomicTimeframeFacet withDatePicker label="Listed within">
+          <AtomicTimeframe unit="hour" />
+          <AtomicTimeframe unit="day" />
+          <AtomicTimeframe unit="week" />
+          <AtomicTimeframe unit="month" />
+          <AtomicTimeframe unit="quarter" />
+          <AtomicTimeframe unit="year" />
+          <AtomicTimeframe unit="year" amount={10} period="next" />
+        </AtomicTimeframeFacet>
+        <AtomicRatingFacet
+          field="ec_rating"
+          label="Rating"
+          numberOfIntervals={5}
+        />
+        <AtomicRatingRangeFacet
+          field="ec_rating"
+          label="Rating Range"
+          numberOfIntervals={5}
+          facetId="ec_rating_range"
+        />
+      </AtomicFacetManager>
+      <AtomicBreadbox />
+      <div className="topbar">
+        <AtomicQuerySummary enableDuration={false} />
+        <AtomicRefineToggle />
+        <AtomicSortDropdown>
+          <AtomicSortExpression label="relevance" expression="relevancy" />
+          <AtomicSortExpression
+            label="Price (low to high)"
+            expression="ec_price ascending"
+          />
+          <AtomicSortExpression
+            label="Price (high to low)"
+            expression="ec_price descending"
+          />
+        </AtomicSortDropdown>
+      </div>
+      <div className="results">
+        <AtomicDidYouMean />
+        <AtomicResultList
+          fields-to-include="ec_price,ec_rating,ec_images,ec_brand,cat_platform,cat_condition,cat_categories,cat_review_count,cat_color"
+          display="grid"
+          image-size="large"
```

Comment:
I just noticed that this is the only component that uses snake case for its props. Would it be possible to change it to camel case?

Additionally, would it be possible to make `fieldsToInclude` (and ResultsPerPage's choicesDisplayed) take arrays as a parameter?

Reply:
Good catch, that was a copy paste mistake. It should be camelCase

As for transforming props that needs an array, from HTML based strings separated by comma, this would require extra preprocessing for every single props like this. And there's quite a lot of them.

It could be cool, but not absolutely mandatory for a first usable release